drivers/bridge/bridge.go

```diff
@@ -70,8 +70,19 @@ type networkConfiguration struct {
 	dbIndex            uint64
 	dbExists           bool
 	Internal           bool
+
+	BridgeIfaceCreator ifaceCreator
 }
 
+// ifaceCreator represents how the bridge interface was created
+type ifaceCreator int8
+
+const (
+	ifaceCreatorUnknown ifaceCreator = iota
+	ifaceCreatedByLibnetwork
+	ifaceCreatedByUser
```

Comment:
Just a minor comment, I would be less verbose
````
const (
    unknown ifaceCreator = iota
    libnetwork
    user
)
```
but that's just my opinion, it already looks good to me.